### PR TITLE
[2 of 5: Icons] Replace PNG sprite with SVG sprite

### PR DESCRIPTION
**Pull request by jfurrow**
~~Depends on #500~~

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @aldipower

File: src/js/components/FilterInputText.js [hunk 1]
```diff
@@ -60,18 +61,23 @@ var FilterInputText = React.createClass({
...
           <a onClick={this.handleClearInput}>
-            <i className={clearIconClasses}></i>
-          </a>
+            <Icon
+              family="mini"
+              id="ring-close"
+              size="mini"
+              className="clickable"
+              color={color} />
+            </a>
```
Comment (kennyt) [inline src/js/components/FilterInputText.js]: this needs to be indented one less

Comment (jfurrow) [inline src/js/components/FilterInputText.js]: Done!

**Comment by jfurrow** [comment]
Integration tests passing locally:

**Pull request merged**